src/napari_omero/plugins/loaders.py

```diff
@@ -13,6 +17,9 @@
 from ..utils import parse_omero_url, timer, lookup_obj, PIXEL_TYPES
 from ..widgets import QGateWay
 
+from dask.cache import Cache
+cache = Cache(2e9)  # Leverage two gigabytes of memory
+cache.register()
```

Comment:
from the commit message, looks like you might just be adding this intentionally for testing.  But just wanted to point out that napari also registers a global cache [here](https://github.com/napari/napari/blob/823b83e39fe89dc30c1002fba7a36160430afbd3/napari/utils/dask_utils.py#L14-L45).  So when `viewer.add_image(data)` eventually gets called, if `data` is a dask array, caching is turned on (starting [here](https://github.com/napari/napari/blob/823b83e39fe89dc30c1002fba7a36160430afbd3/napari/layers/base/base.py#L140))